client-app/src/core/constants.ts

```diff
@@ -4,8 +4,8 @@ import {  BvTableFieldArray } from 'bootstrap-vue';
 export const storeName = window.STORE_NAME;
 export const locale = window.LOCALE;
 // Need to trim store and language from base URL because they will be added later as parameters for each API call.
-export const baseUrl = window.BASE_URL.replace(`/${storeName}`, "/")
-  .replace(`/${locale}`, "/")
+export const baseUrl = window.BASE_URL.replace( new RegExp(`(?<!http:/)/${storeName}`), "/")
+  .replace(new RegExp(`(?<!http:/)/${locale}`), "/")
```

Comment:
@trueboroda It looks like a hack, but if it works then OK :)
In my opinion it is better to use built-in functions of URL transformations

Reply:
Yes, we can use URL parsing for this purpose. It was done.